### DISCUSSION TITLE
Self-hosted Jitsi Meet errors 200 on colibri-ws

### DISCUSSION THREAD

**Discussion by markblumenau** [jbg/gst-meet#58] [General]
Hello, 
I am trying to use gst-meet with a self-hosted instance of Jitsi Meet. I enabled (or at least I think so) colibri-ws in sip-communicator. However, I get 
```
2022-09-27T22:45:48.128388Z  WARN Failed to connect Colibri WebSocket, will retry: Http(Response { status: 200, version: HTTP/1.1, headers: {"server": "nginx/1.18.0 (Ubuntu)", "date": "Tue, 27 Sep 2022 22:45:48 GMT", "content-type": "text/html", "transfer-encoding": "chunked", "connection": "keep-alive", "vary": "Accept-Encoding", "strict-transport-security": "max-age=63072000"}, body: None })
```
Looks like there is some kind of trouble with nginx or jvb config. Can someone tell me how to enable colibri-ws in configs in order to use gst-meet?
Thanks in advance.

**Comment by jbg** [comment]
The response for the Colibri websocket should be `101 Switching Protocols` and not `200 OK`. If you are proxying Colibri WS through nginx (as in the "standard" Jitsi Meet config), then it's likely a problem with your nginx config.

It presumably isn't working correctly in the browser either? (You will see warnings/errors in the JS console about the Colibri WebSocket and Colibri functionality won't work correctly.) Since the problem isn't related to gst-meet, the right place to get help with fixing your Colibri websocket would be the [Jitsi community forums](https://community.jitsi.org).

**Reply by markblumenau** [reply]
Yes, it was a nginx issue after all. Also, for whatever reason jvb was listening on ipv6. Thanks!
